src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -78,7 +78,7 @@ private static void createBuses(List<Bus> buses, LfNetworkParameters parameters,
         }
     }
 
-    private static void createVoltageControls(List<LfBus> lfBuses, LfNetworkParameters parameters) {
+    private static void createVoltageControls(List<LfBus> lfBuses, LfNetworkParameters parameters, ReportNode reportNode) {
```

Comment:
here and in many places after, don't add this argument in method signature. Instead, you should be able to get the LfNetwork report node by doing `someBus.getNetwork().getReportNode()`